### ISSUE TITLE
Ability to display livechat in PeerTube embeds

### ISSUE THREAD

**Issue by Chocobozzz** [JohnXLivingston/peertube-plugin-livechat#92]
Hi :)

It would be interesting to add ability for users to display livechat in PeerTube embed, using `embed` scope.

**Comment by JohnXLivingston** [comment]
Hello :)
Plugins can now be added in embed?? Nice!

I'll add this on my TODO list. Hope the iframe is not to small for both video and chat. 

I can also add an option in my «share» popup: 
![image](https://user-images.githubusercontent.com/38844060/153378159-5ab3b1d1-51ed-41ce-aa41-6e9c6e572142.png)
For example, a checkbox «Embedded chat». So the user can choose to embed the video and the chat in two separate iframes.

**Comment by Chocobozzz** [comment]
Yes, or create a videojs component to display tchat as an overlay. I'm not sure what would be the best/simpler way.

Awaiting this feature, is there an available link so people can embed the chat below the peertube embed (using 2 iframes then)

**Comment by JohnXLivingston** [comment]
> Awaiting this feature, is there an available link so people can embed the chat below the peertube embed (so using 2 iframes)?

I think the link in the share popup should work. Unless there is a cross domain issue (I have no time to test before next week, I'm currently very busy on other projects).
